src/commands/mod.rs

```diff
@@ -324,6 +324,7 @@ impl DaemonControl {
             },
             rescan_progress,
             timestamp: db_conn.timestamp(),
+            last_poll_timestamp: db_conn.last_poll_timestamp(),
```

Comment:
seems to me that we do two database call to get db_wallet for timestamp() and last_poll_timestamp(), can we have instead wallet = db_conn.wallet() and then wallet.timestamp and wallet.last_poll_timestamp

Reply:
Yes I think that would be better. I'll add a new `wallet()` method to the DB interface.